tests/functional/spec/thank_you.spec.js

```diff
@@ -0,0 +1,41 @@
+import { SubmitPage } from "../base_pages/submit.page.js";
+import HubPage from "../base_pages/hub.page";
+import CheckboxPage from "../generated_pages/title/single-title-block.page";
+import ThankYouPage from "../base_pages/thank-you.page";
+
+describe("Thank You Social", () => {
+  describe("Given I launch a social theme schema", () => {
```

Comment:
We usually go with questionnaire here rather than schema. Possible themed aswell

Reply:
Changed to "... themed questionnaire", thanks!